src/Installers/Windows/SharedFrameworkLib/Library.wxs

```diff
@@ -11,13 +11,13 @@
             <?undef DotNetHome?>
             <?endif?>
             <?if $(var.Platform)=x86?>
-            <?define SharedFrameworkInstallCondition=(NOT OPT_NO_SHAREDFX) AND (NOT OPT_NO_X86)?>
+            <?define SharedFrameworkInstallCondition=(NOT OPT_NO_SHAREDFX OR OPT_NO_SHAREDFX="0") AND (NOT OPT_NO_X86)?>
```

Comment:
Why doesn't `OPT_NO_X86` need a check for `0`❔ Also, why haven't all options been set to `0` or `1` by this point❔

Reply:
Variables can't be assigned conditionally in bundles, only .msi's. When the hosting bundle calls into these libraries, the variables could still be empty if this is the first time the bundle is installed, and the user passed nothing from the command line. Good catch on OPT_NO_X86 though, fixing.